bedrock/foundation/templates/foundation/trademarks/distribution-policy.html

```diff
@@ -13,11 +13,15 @@
 {% block article_content %}
   <p>
     {% trans license=url('mozorg.mpl.index') %}
-      What you can and cannot do with our software products and code – independent of our trademarks – is governed by the relevant software licenses. For example, most Firefox code is released under the <a href="{{ license }}">Mozilla Public License</a> version 2.0. The Mozilla Public License does not contain grant permission to use Mozilla trademarks, as is set out in its section 2.1. "Grants":
+      What you can and cannot do with our software products and code – independent of our trademarks – is governed by the relevant software licenses. For example, most Firefox code is released under the <a href="{{ license }}">Mozilla Public License</a> version 2.0.
     {% endtrans %}
   </p>
 
-  <blockquote cite="{{ url('mozorg.mpl.2.0.index') }}">
+  <p>
+  {{_('The Mozilla Public License does not contain grant permission to use Mozilla trademarks, as is set out in its section 2.1. "Grants":')}}
+  </p>
+
+  <blockquote class="non-standout-quote" cite="{{ url('mozorg.mpl.2.0.index') }}">
```

Comment:
Nit: perhaps this could be `t-small`? to modify the regular blockquote styling?

Reply:
Same as above - as long as the copy looks roughly the same happy to defer.